src/dev/sass/run_build_sass_cli.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { run } from '../run';
+import { REPO_ROOT } from  '../constants';
+import { buildSass } from './build_sass';
+
+run(async ({ log, flags: { kibanaDir } }) => {
```

Comment:
Since this is async, will the script still exit and allow a process to follow?  I wouldn't want Storybook to start until this completes.

Reply:
Yep, `run()` executes the async function and reflects its success/failure with the process exit code, so if this fails it will exit with a 1, and if it succeeds the process will exit with a 0